### ISSUE TITLE
new CI issues

### ISSUE THREAD

**Issue by willemneal** [AssemblyScript/assemblyscript#863]
I currently have the following error when pushing trying to test the new setup.  
```
Author 'willemneal@users.noreply.github.com' does not appear to be listed in the NOTICE file, yet.
```

I first pushed with gitkraken, which is logged into my github account.  But then tried normally and it still didn't like it.

**Comment by dcodeIO** [comment]
Ah, I've made the check non-optional in the workflow to check what happens. I guess if the email isn't public on ones GitHub profile, this is what happens. Two ways to deal with it: Either make it optional again, but not sure how useful it is then, or require contributors to have a public email address.

**Comment by dcodeIO** [comment]
Well-actually-ing myself: I have a different email address as my public one, but it still works. So it might also be a Git configuration issue of the kind

```
git config --global user.name "username"
git config --global user.email "email"
```

**Comment by willemneal** [comment]
I tried exactly that, but I can try to make my email address public.

**Comment by dguo** [comment]
I ran into an issue with this check when I created #919. I have two email addresses in GitHub: one Gmail and one personal domain. I created the commit with my personal domain address, but the NOTICE check had an error because it couldn't find my Gmail address. I added a `git log` to see what was going on, and [the logs](https://github.com/AssemblyScript/assemblyscript/commit/a2434697a76f5a3f74cb5c6ccf089fdbb39de8f2/checks?check_suite_id=272482461) show that it was checking the merge commit, which had my Gmail address because it's my primary one in GitHub.

To get around the issue, I changed my primary to my personal domain address, but maybe this is the root cause of @willemneal's issue. Maybe the check should ignore the merge commit?
